### PR TITLE
add is_view field to TableDetailAPI payload

### PR DESCRIPTION
**Pull request by youngyjd**
This PR is to add `is_view` field to TableDetailAPI payload, so frontend can get the value and differentiate Views from Tables.

### CODE REVIEW

File: metadata_service/proxy/neo4j_proxy.py [hunk 1]
```diff
@@ -81,7 +81,8 @@ def get_table(self, *, table_uri: str) -> Table:
                       watermarks=wmk_results,
                       table_writer=table_writer,
                       last_updated_timestamp=timestamp_value,
-                      source=source)
+                      source=source,
+                      is_view=last_neo4j_record['tbl']['is_view'])
```
Comment (jinhyukchang) [inline metadata_service/proxy/neo4j_proxy.py]: Would this break if the underlying node attribute does not have `is_view`?

You may use https://github.com/lyft/amundsenmetadatalibrary/blob/master/metadata_service/proxy/neo4j_proxy.py#L227

Comment (youngyjd) [reply metadata_service/proxy/neo4j_proxy.py]: Good catch! 👍

**Comment by youngyjd** [comment]
@feng-tao @jinhyukchang PTAL

**Comment by youngyjd** [comment]
🔒

**Comment by youngyjd** [comment]
🔓

File: metadata_service/proxy/neo4j_proxy.py [hunk 2]
```diff
@@ -69,6 +69,10 @@ def get_table(self, *, table_uri: str) -> Table:
 
         wmk_results, table_writer, timestamp_value, owners, tags, source = self._exec_table_query(table_uri)
 
+        # neo4j returns is_view (boolean data type) as str, so we need to convert it to bool
+        is_view = self._safe_get(last_neo4j_record, 'tbl', 'is_view')
+        is_view = is_view.lower() in ['t', 'true'] if is_view else False
```
Comment (feng-tao) [inline metadata_service/proxy/neo4j_proxy.py]: is there any better way?

**Comment by youngyjd** [comment]
@feng-tao @jinhyukchang Please take a look

**Pull request merged**